src/functions/Get-TypeContent.ps1

```diff
@@ -0,0 +1,65 @@
+function Get-TypeContent {
+  <#
+  .SYNOPSIS
+    Return the text for a Puppet Resource API Type given a DSC Resouce.
+  .DESCRIPTION
+    Return the text for a Puppet Resource API Type given a DSC Resouce.
+    It will return the text but _not_ directly write out the file.
+  .PARAMETER DscResource
+    A DSCResourceInfo object with the required parameter information
+    retrieved - if the object does _not_ already have the updated info
+    for the parameters, this function will attempt to retrieve it.
+  .EXAMPLE
+    Get-DscResourceTypeInformation -Name PSRepository | Get-TypeContent
+
+    This will retrieve a DSC resource from the PSModulePath, retrieve the information
+    needed to represent the DSC resource's properties as Puppet Resource API type
+    attributes and then return the representation of that DSC resource appropriate
+    for the Puppet Resource API.
+  #>
+  [cmdletbinding()]
+  param (
+    [OutputType([String], [String[]])]
+    [Parameter(ValueFromPipeline)]
+    [Microsoft.PowerShell.DesiredStateConfiguration.DscResourceInfo[]]$DscResource
+  )
+
+  Begin {}
```

Comment:
Do we need this if there is no empty end{}?

Reply:
🤔 I actually don't know if you can skip them altogether... I guess you can? Whoops.